src/main/resources/lib/vsphere/blockWrapper.jelly

```diff
@@ -0,0 +1,16 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:d="jelly:define">
+    <!-- TODO remove and switch to div after baseline is 2.264 or newer -->
+    <j:choose>
+        <j:when test="${divBasedFormLayout}">
+            <div>
+                <d:invokeBody/>
+            </div>
+        </j:when>
+        <j:otherwise>
+            <table style="width:100%">
+                <d:invokeBody/>
+            </table>
+        </j:otherwise>
+    </j:choose>
+</j:jelly>
```

Comment:
I presume that this is the core of the fix?
i.e. pre 2.264 it uses `table` and post it uses `div`?

...and does it do this automatically?  Is that what `test="${divBasedFormLayout}"` achieves?

Reply:
Yes it’s a property on set on >=2.264